internal/apis/certmanager/validation/certificaterequest_test.go

```diff
@@ -826,39 +826,3 @@ func mustGenerateCSR(t *testing.T, crt *cmapi.Certificate) []byte {
 
 	return csrPEM.Bytes()
 }
-
-func Test_patchDuplicateKeyUsage(t *testing.T) {
```

Comment:
we now validate by looking at the generated template. These templates already have these usages deduplicated.